src/Interface/Modules/Render/ES/SRInterface.cc

```diff
@@ -566,7 +566,7 @@ void SRInterface::runGCOnNextExecution()
       updateCamera();
       updateWorldLight();
 
-      mCore.execute(0, 50);
+      mCore.execute(0, 0);
```

Comment:
After digging into this a bit more, this is the only hard-coded number case of calling mCore.execute. Instead of passing magic numbers and changing said numbers, make a new function to wrap this special call--give it a name that explains why it's now 0,0, if you like. Something like `executeWithNoDelayToDetermineSelectedWidget`

Reply:
As discussed today: remove first parameter entirely, then add the new function with expressive name.